src/platform/plugins/shared/dashboard/server/dashboard_saved_object/dashboard_saved_object.ts

```diff
@@ -79,6 +79,10 @@ export const createDashboardSavedObjectType = ({
       },
       optionsJSON: { type: 'text', index: false },
       panelsJSON: { type: 'text', index: false },
+      sections: {
+        properties: {},
+        dynamic: false,
+      },
```

Comment:
AFAIK, if we don't need to query/aggregate by this field, there is no need to define it in the mappings.

Reply:
oh! I missed that! All good then! Thanks for explaining! 🧡